frontend/src/components/atoms/Helper/style.css

```
@@ -8,6 +8,12 @@
   vertical-align: middle;
 }
 
+.helper-large::after {
```

ne fonctionne pas chez moi. Je pense qu'il faut être plus précis sur le sélecteur CSS pour augmenter la priorité de la règle et évité qu'elle soit override par d'autres règles
```suggestion
.helper.helper-large::after {
```